wallet/backend.go

```diff
@@ -0,0 +1,66 @@
+package wallet
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io"
+	"perun.network/go-perun/wallet"
+)
+
+// SignatureLength is the length of valid Cardano signatures in bytes
+const SignatureLength = 64
+
+// RemoteBackend is a wallet.Backend implementation with a remote server for signing data and verifying signatures
+type RemoteBackend struct {
+	walletServer Remote
+}
+
+// MakeRemoteBackend a new RemoteBackend struct, setting the wallet server appropriately
+func MakeRemoteBackend(remote Remote) RemoteBackend {
+	return RemoteBackend{remote}
+}
+
+// NewAddress returns a pointer to a new empty PubKey
```

Comment:
I agree to renaming it to `Address`

Regarding the latter: I think we should talk about what exactly `Address` should be soon.
In my opinion the "final" version of `Adress` should look like this:
- String() returns something like `addr1vpu5vlrf4xkxv2qpwngf6cjhtw542ayty80v8dyr49rf5eg0yu80w`
- There should be a smart constructor creating (or filling) an addres from `addr1vpu5vlrf4xkxv2qpwngf6cjhtw542ayty80v8dyr49rf5eg0yu80w`
- the binary representation (`(Un)MarshalBinary`) should probably be whatever an address looks like in the outputs of plutus transactions (The binary representation I'm least sure of... it seems weird. Another options would be the byte representation of the address string `addr1vpu5vlrf4xkxv2qpwngf6cjhtw542ayty80v8dyr49rf5eg0yu80w` - maybe that is even how addresses look on-chain?)

Still, for the current version I am fine with Addresses being just public keys and leaving out the smart constructor from `addr1vpu5vlrf4xkxv2qpwngf6cjhtw542ayty80v8dyr49rf5eg0yu80w` to avoid complex encoding/decoding. Renaming it now will make changing this in the future feel more comfortable though! Do you agree with that?